samples/dashboards/demo/live-weather/demo.html

```diff
@@ -6,6 +6,7 @@
 <script src="https://code.highcharts.com/modules/solid-gauge.js"></script>
 <script src="https://code.highcharts.com/modules/accessibility.js"></script>
 <script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/layout.js"></script>
```

Comment:
This demo doesn't use edit mode, layout.js should not be necessary.

Reply:
You setup the gui, which is a part of `layout.js`

![Screenshot 2024-02-12 at 15 26 32](https://github.com/highcharts/highcharts/assets/2115686/a7c4b0b1-8d9b-4833-acf5-9e935cffc335)